app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,7 @@
       <%= render 'shared/nav' %>
     <% end %>
 
-    <div class="application container <%= 'extra-padding' unless show_sub_nav?(@school, @hide_subnav) %>">
+    <div class="application container<%= ' extra-padding' unless show_sub_nav?(@school, @hide_subnav) %><%= ' header-fix' if header_fix_enabled? %>">
```

Comment:
perhaps this could be extracted to a helper? something like 
```
<div class="application container <%= conditional_application_container_classes %>">
```
```
def conditional_application_container_classes
  classes = ''
  classes += ' extra-padding' unless show_sub_nav?(@school, @hide_subnav)
  classes += ' header-fix' if header_fix_enabled?
  classes
end
```